### DISCUSSION TITLE
meet link

### DISCUSSION THREAD

**Discussion by frenkhub** [InteractionDesignFoundation/add-event-to-calendar-docs#43] [Q&A] [answered]
Is it possible to set a meet link too? Thanks.

**Comment by alies-dev** [comment, marked as answer]
Yes: `vcon=meet`.

```
https://calendar.google.com/calendar/render?action=TEMPLATE&text=Test&dates=20261231T190000Z/20261231T200000Z&vcon=meet
```

Verified on 2026-08-19: the editor opens with a real Meet link already generated, for example `meet.google.com/xxx-xxxx-xxx`, together with the usual conference controls.

Two limitations worth knowing:

* the value is compared against the literal string `meet`, so no other conferencing provider can be requested this way;
* it always creates a **new** conference. There is no parameter that attaches an existing Meet URL to the event.

Now documented in `services/google.md`.
